ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -146,3 +152,49 @@ If.HdmiIec61937 {
 	}
 	True.Macro.save_hdmi_cfg.HdmiPCMSave { Name "42-sof-hdmi" }
 }
+
+If.multi_init {
+	Condition {
+		Type String
+		Empty "${var:MultiCodec1} "
+	}
+	False {
+		Include.multi_init.File "/codecs/${var:MultiCodec1}/init.conf"
+	}
+}
+
+If.multi_speaker {
```

Comment:
Those sections must be moved before ????Codec1 variables use to avoid multiple init.conf inclusions. So before "If.hs_init", "If.mic_init".